linguatec_lexicon/templates/linguatec_lexicon/importdata.html

```diff
@@ -0,0 +1,76 @@
+{% extends 'rest_framework/base.html' %}
+{% load i18n static %}
+
+{% block title %}{{ title }}{% endblock %}
+
+{% block branding %}
+<a class="navbar-brand" rel="nofollow" href="/">{{ title }}</a>
+{% endblock %}
+
+{% block content %}
+<h1>{{ title_data }}</h1>
+<div class="row">
+    <div class="col-12">
+        <div>
+            Selecciona el fichero XLSX para importar las variaciones diatopicas.
+        </div>
+        <div class="data-validator-form">
+            <form action="" enctype="multipart/form-data" method="post">
+                {% csrf_token %}
+                {{ form.as_p }}
+                <label for="lexicon_code">Codigo lexicon</label><input type="text" name="lexicon_code" /></br>
+                </br>
+                <input name="type_import" type="hidden" value="data">
+                <input id="Importar" type="submit" value="Importar" />
+            </form>
+        </div>
+    </div>
+</div>
+<div class="row">
+    <h3><strong>{{ is_finished }}</strong></h3>
+</div>
+
+<h1>{{ title_variation }}</h1>
+<div class="row">
+    <div class="col-12">
+        <div>
+            Selecciona el fichero XLSX para importar las variaciones diatopicas.
+        </div>
+        <div class="data-validator-form">
+            <form action="" enctype="multipart/form-data" method="post">
+                {% csrf_token %}
+                {{ form.as_p }}
+                <label for="lexicon_code">Codigo lexicon</label><input type="text" name="lexicon_code" /></br>
```

Comment:
Código (tilde missing)
Leave a blank space between Código and input.

Reply:
Fixed